src/applications/simple-forms/20-10207/tests/e2e/fixtures/mocks/featureToggles.json

```diff
@@ -0,0 +1,6 @@
+{
+  "data": {
+    "type": "feature_toggles",
+    "features": [{ "name": "profile_show_profile_2.0", "value": false }]
```

Comment:
Not right now, but I thought I'd add it now.  It'll be needed as I build more pages, and I always leave Profile 2.0 ON just in case prefill chokes w/o it.